src/app/home/home.html

```diff
@@ -1 +1,2 @@
 <h1>Home</h1>
+<device-list></device-list>
```

Comment:
can device-list be moved to a separate route?
(it might be `/` root route, though I want it to be configurable)